### PR TITLE
Use testproject from keboola/go-utils

### PR DESCRIPTION
**Pull request by webrouse**
Changes:
- Part of `testproject.Project` has been replaced by `go-utils/pkg/testproject` package.

### CODE REVIEW

File: internal/pkg/api/client/storageapi/token_test.go [hunk 1]
```diff
@@ -30,22 +30,22 @@ func TestGetToken(t *testing.T) {
 	t.Parallel()
 	project := testproject.GetTestProject(t, env.Empty())
 	logger := log.NewDebugLogger()
-	api := New(context.Background(), logger, project.StorageApiHost(), false)
+	api := New(context.Background(), logger, project.StorageAPIHost(), false)
```
Comment (webrouse) [inline internal/pkg/api/client/storageapi/token_test.go:33]: Acronyms sa v Go pisu vzdy velkym, v tomto projekte sme to neriesili, ale v API klientovi je to uz spravne (kontroluje to aj linter):
`The rationale for uppercase abbreviations in Go symbol name derives from the english convention for writing acronyms using all uppercase letters`

File: internal/pkg/env/map.go [hunk 2]
```diff
@@ -44,6 +44,14 @@ func FromOs() (*Map, error) {
 	return m, nil
 }
 
+func (m *Map) Clone() *Map {
```
Comment (webrouse) [inline internal/pkg/env/map.go:47]: Pridane klonovanie `env.Map`

File: internal/pkg/state/load_remote_test.go [hunk 3]
```diff
@@ -461,5 +461,5 @@ func loadRemoteState(t *testing.T, m *manifest.Manifest, projectStateFile string
 	filter := m.Filter()
 	_, localErr, remoteErr := state.Load(LoadOptions{RemoteFilter: filter, LoadRemoteState: true})
 	assert.NoError(t, localErr)
-	return state, envs, remoteErr
+	return state, testProject.Env(), remoteErr
```
Comment (webrouse) [inline internal/pkg/state/load_remote_test.go:464]: Do `GetTestProject(t, envs)` vstupuju premenne prostredia ... doteraz to bolo tak, ze sa k tymto env pridavali dalsie ohladom testovacieho projektu ... nebolo to velmi intuitivne, ... ze tam vlozis `envs` a oni sa zmenia.

Teraz je to uz immutable.

File: internal/pkg/utils/testproject/project.go [hunk 4]
```diff
@@ -33,64 +31,52 @@ import (
 )
 
 type Project struct {
```
Comment (webrouse) [inline internal/pkg/utils/testproject/project.go:33]: Cast `Project`, ohladom exkluzivneho pristupu, bola presunuta do `go-utils`.
Tu zostalo vymazanie vsetkeho z projektu, este so starymi APIs.

**Review by JakubMatejka — APPROVED** [review]

**Pull request merged**